edge/services/helloSecretWorld/CreateService.md

```diff
@@ -0,0 +1,262 @@
+\<secret-value\> is the value of the secret as stated in the `hw_who` file.
+
+  on **Linux**:
+
+  ```bash
+  eval $(hzn util configconv -f horizon/hzn.json)
+  sudo tail -f /var/log/syslog | grep ${SERVICE_NAME}[[]
+  ```
+
+  on **Mac**:
+
+  ```bash
+  eval $(hzn util configconv -f horizon/hzn.json)
+  sudo docker logs -f $(sudo docker ps -q --filter name=$SERVICE_NAME)
+  ```
+
+11. Stop the service:
+
+  ```bash
+  hzn dev service stop
+  ```
+
+12. You are now ready to publish your edge service and pattern, so that they can be deployed to real edge nodes. Instruct Horizon to push your docker image to your registry and publish your service in the Horizon Exchange:
+
+  ```bash
+  hzn exchange service publish -f horizon/service.definition.json
+  hzn exchange service list
+  ```
+
+13. (This step must be done by an organization admin) Create an organization-wide secret called `secret-name` in your organization's secrets manager. The flags `--secretKey` and `--secretDetail` provide the key and value for the secret, respectively. The key is ignored by the service, but `"Hello <secret-value>!"` will be printed by the service.
+
+  ```bash
+  hzn sm secret add --secretKey <secret-key> --secretDetail <secret-value> secret-name
+  ```
+
+14. Edit your pattern definition file to make the pattern not public, then publish your edge node deployment pattern in the Horizon Exchange:
+
+  ```bash
+  jq '.public = false' horizon/pattern.json > horizon/pattern.tmp && mv horizon/pattern.tmp horizon/pattern.json
+  hzn exchange pattern publish -f horizon/pattern.json
+  hzn exchange pattern list
+  ```
+Horizon will check for the existence of `secret-name` in your organization's secrets manager. If it is not found in the secrets manager, Horizon will not be able to publish the pattern to the exchange.
+
+15. Register your edge node with Horizon to use your deployment pattern:
+
+  ```bash
+  hzn register -p pattern-${SERVICE_NAME}-$(hzn architecture) -s $SERVICE_NAME --serviceorg $HZN_ORG_ID
+  ```
+
+16. **Open another terminal** and view the service output with the "follow" flag:
+
+  ```bash
+  hzn service log -f $SERVICE_NAME
+  ```
+  
+17. (This step must be done by an organization admin) Update the secret `secret-name` in the secrets manager using the Horizon CLI. To see any change reflected in the output, the argument passed to `--secretDetail` should be different from the initial secret creation command, so that the value of the secret can change.
+
+  ```
+  hzn sm secret add --secretKey <secret-key> --secretDetail <new-secret-value> secret-name
+  ```
+
+A prompt will pop up in the terminal asking you to confirm that you want to overwrite the existing `secret-name` in the secrets manager. Enter `y` to confirm.
+
+18. After some time you should see the output of the service change to **\<your-node-id> says: Hello \<new-secret-value>!**, where `<new-secret-value>` is the new value of the secret as updated in the secrets manager in Step 17.
```

Comment:
I think we need to reword this to be something generic like: Now your service should be able to observe the change in the secret.
They may have changed the service to not display the message any more.